bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -0,0 +1,954 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::error::Error;
+
+use bee_ledger::types::Receipt;
+use bee_message::{
+    address::{Address, Ed25519Address, ED25519_ADDRESS_LENGTH},
+    input::{Input, TreasuryInput, UtxoInput},
+    milestone::MilestoneIndex,
+    output::{Output, SignatureLockedDustAllowanceOutput, SignatureLockedSingleOutput, TreasuryOutput},
+    parents::Parents,
+    payload::{
+        indexation::IndexationPayload,
+        milestone::{
+            MilestoneId, MilestonePayload, MilestonePayloadEssence, MILESTONE_MERKLE_PROOF_LENGTH,
+            MILESTONE_PUBLIC_KEY_LENGTH,
+        },
+        receipt::{MigratedFundsEntry, ReceiptPayload, TailTransactionHash, TAIL_TRANSACTION_HASH_LEN},
+        transaction::{Essence, RegularEssence, TransactionId, TransactionPayload},
+        treasury::TreasuryTransactionPayload,
+        Payload,
+    },
+    signature::{Ed25519Signature, SignatureUnlock},
+    unlock::{ReferenceUnlock, UnlockBlock, UnlockBlocks},
+    Message, MessageBuilder, MessageId,
+};
```

Comment:
looks like you could make good use the `prelude` of bee-message

Reply:
good idea, let's do that in with the second PR then